types/node/http2.d.ts

```diff
@@ -583,7 +583,7 @@ declare module "http2" {
     }
 
     export class Http2ServerRequest extends stream.Readable {
-        private constructor();
+        constructor(stream: ServerHttp2Stream, headers: string[], options: stream.ReadableOptions, rawHeaders: string);
```

Comment:
I think `headers` should be of type `IncomingHttpHeaders` and `rawHeaders` should be `string[]`. According to source `options` may be also `undefined`.

Reply:
thanks, fixed.